$OpenBSD$

Relax SimpleX Chat bounds for GHC 9.10 and current SimpleXMQ dependencies.

--- simplex-chat.cabal.orig
+++ simplex-chat.cabal
@@ -303,8 +303,8 @@
     , base >=4.7 && <5
     , base64-bytestring >=1.0 && <1.3
     , composition ==1.0.*
-    , constraints >=0.12 && <0.14
-    , containers ==0.6.*
+    , constraints >=0.12 && <0.15
+    , containers >=0.6 && <0.8
     , crypton ==0.34.*
     , crypton-x509 ==1.7.*
     , data-default ==0.7.*
@@ -314,7 +314,7 @@
     , file-embed ==0.0.15.*
     , filepath ==1.4.*
     , http-types ==0.12.*
-    , http2 >=4.2.2 && <4.3
+    , http2 >=5.1 && <5.4
     , memory ==0.18.*
     , mtl >=2.3.1 && <3.0
     , network >=3.1.2.7 && <3.2
@@ -328,7 +328,7 @@
     , stm ==2.5.*
     , terminal ==0.2.*
     , time ==1.12.*
-    , tls >=1.9.0 && <1.10
+    , tls >=2.0 && <2.3
     , unliftio ==0.2.*
     , unliftio-core ==0.2.*
     , uri-bytestring >=0.3.3.1 && <0.4
@@ -350,9 +350,9 @@
       , sqlcipher-simple ==0.4.*
   if impl(ghc >= 9.6.2)
     build-depends:
-        bytestring ==0.11.*
+        bytestring >=0.11 && <0.13
       , process ==1.6.*
-      , template-haskell ==2.20.*
+      , template-haskell >=2.20 && <2.23
       , text >=2.0.1 && <2.2
   if impl(ghc < 9.6.2)
     build-depends:
@@ -462,7 +462,7 @@
     , simplexmq >=6.3
     , stm ==2.5.*
     , unliftio ==0.2.*
-    , websockets ==0.12.*
+    , websockets >=0.12 && <0.14
   default-language: Haskell2010
   if flag(client_postgres)
     cpp-options: -DdbPostgres
@@ -502,7 +502,7 @@
     , base >=4.7 && <5
     , base64-bytestring >=1.0 && <1.3
     , composition ==1.0.*
-    , containers ==0.6.*
+    , containers >=0.6 && <0.8
     , crypton ==0.34.*
     , directory ==1.3.*
     , filepath ==1.4.*
@@ -532,7 +532,7 @@
         sqlcipher-simple ==0.4.*
   if impl(ghc >= 9.6.2)
     build-depends:
-        bytestring ==0.11.*
+        bytestring >=0.11 && <0.13
       , text >=2.0.1 && <2.2
   if impl(ghc < 9.6.2)
     build-depends:
@@ -624,7 +624,7 @@
     , base >=4.7 && <5
     , base64-bytestring >=1.0 && <1.3
     , composition ==1.0.*
-    , containers ==0.6.*
+    , containers >=0.6 && <0.8
     , crypton ==0.34.*
     , deepseq ==1.4.*
     , directory ==1.3.*
@@ -661,7 +661,7 @@
         sqlcipher-simple ==0.4.*
   if impl(ghc >= 9.6.2)
     build-depends:
-        bytestring ==0.11.*
+        bytestring >=0.11 && <0.13
       , hspec ==2.11.*
       , process ==1.6.*
       , text >=2.0.1 && <2.2
